src/patternfly/demos/Toolbar/toolbar-template.hbs

```diff
@@ -25,6 +26,13 @@ Options: [
       {{!-- toggle group --}}
       {{#if toolbar-template--HasToggleGroup}}
         {{#> toolbar-group toolbar-group--modifier="pf-m-toggle-group pf-m-show-on-xl"}}
+          {{#if toolbar-template--HasExpandAll}}
```

Comment:
I think you'll want to move this outside of the `toggle-group` for two reasons. 1) I _think_ the expand all button should remain visible when `toggle-group` hides its contents and 2) You would need to consider it in the expandable content section when `toggle-group` moves it there. 

@mmenestr @mcarrano does this sound right to you? Would you expect the expand all button to remain visible when filters are hidden?

Reply:
@mattnolting good call, updated!

Reply:
What do you mean when filters are hidden? Like if there's no toolbar?